style.css

```diff
@@ -132,8 +141,511 @@ header {
+  height: 1px;
+  background-image: url(img/pro5.png);
+  background-color: rgb(208, 204, 204);
+  margin-bottom: 30px;
+}
+
+/* == professional section start == */
+
+.professional-card {
+  margin-bottom: 24px;
+  display: grid;
+  grid-template-columns: repeat(1, 1fr) / 1;
+  flex-wrap: wrap;
+  place-items: center;
+  width: 375px;
+}
+
+.card2 {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+  width: 327px;
+  background:
+
+    linear-gradient(
+      180.45deg,
+      rgba(38, 38, 38, 0) 0.75%,
+      rgba(38, 38, 38, 0.9) 61.94%
+    ),
+    url(img/Img\ Placeholder55.png);
+}
+
+.card-title {
+  margin-top: 84px;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  width: 295px;
+  height: 81px;
+  text-align: center;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.card-pro-text {
+  color: #fff;
+  align-items: center;
+  width: 295px;
+  margin-left: 15px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.card-pro-buttons ul {
+  margin-bottom: 14px;
+  width: 327px;
+  display: flex;
+  list-style: none;
+  padding-top: 22px;
+}
+
+.pro-card-button {
+  width: 327px;
+  background-color: #ff6b00;
+  color: white;
+  margin-top: 24px;
+  padding: 16px 0 4px 0;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  text-decoration: none;
+  height: 40px;
+}
+
+.pro-card-button:hover {
+  width: 327px;
+  background: #fff5e1;
+  color: #ff6b00;
+  margin-top: 24px;
+  padding: 16px 0 4px 0;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  text-decoration: none;
+  border: 1px solid #ff6b00;
+}
+
+/* === about section start === */
+.card-3 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 120px;
+  padding-top: 120px;
+  width: 375px;
+  background: #1c1a19;
+}
+
+.card-about-title h3 {
+  width: 327px;
+  margin-bottom: 24px;
+  margin-left: 20px;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.card-about-text p {
+  width: 327px;
+  margin-left: 18px;
+  margin-bottom: 12px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: lighter;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.skill {
+  margin-top: 107px;
+  display: grid;
+  place-items: center;
+  width: 327px;
+  order: 2;
+}
+
+.skill-group {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+}
```

Comment:
- [ ] :thinking: let's use another font family here to match the Figma design, 'Inter' is the right one.
![Screenshot from 2023-01-20 19-10-41](https://user-images.githubusercontent.com/97834160/213765281-15453017-9acc-4c6b-b9e3-6b67a982734b.png)